DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -48,6 +48,11 @@ public IActionResult Index(LoginViewModel model)
                 return View("Index", model);
             }
 
+            if (unverifiedAdminUser != null && unverifiedDelegateUsers.Count == 0)
+            {
+                unverifiedDelegateUsers = userService.GetDelegateUsersByUsername(unverifiedAdminUser.EmailAddress);

```

Comment:
Could we just fetch that as part of the first call to the users service (we might need to change the name of the method). Then the user service could:
- Get admin by username
- If admin exists, get delegates by admin.emailAddress
- If not, get delegates by username
This would also reduce the amount of calls to db.

Reply:
Yes, that makes a lot more sense.